rfcs/rfc-009-staking.md

```diff
@@ -0,0 +1,59 @@
+
+### Rewards
+Rewards are expected to be payed out to active validators as a function of validator activity and as a proportion of the percentage of SOL they have at stake out of the entirety of the staking pool.
+
+We expect to define a baseline annual validator payout/inflation rate based on the total SOL deposited. E.g. 10% annual interest on SOL deposited with X total SOL deposited as slashable on network. This is the same designed as currently proposed in Casper FFG which has additionally specified how inflation rates adjust as a function of total ETH deposited. Specifically, Casper validator returns are proportional to the inverse square root of the total deposits and initial annual rates are estimated as:
+
+| Deposit Size | Annual Validator Interest |
+|--------------|---------------------------|
+| 2.5M ETH     | 10.12%                    |
+| 10M ETH      | 5.00%                     |
+| 20M ETH      | 3.52%                     |
+| 40M ETH      | 2.48%                     |
+
+This has the nice property of potentially incentivizing participation around a target deposit size. Incentivisation of specific participation rates more directly (rather than deposit size) may something also worth exploring.
+
+The specifics of the Solana validator reward scheme are to be worked out in parallel with a design for transaction fee assignment as well as our storage mining reward scheme.
+
+### Slashing rules
+Unlike proof-of-work where off-chain capital expenses are already deployed at the time of block construction/voting, proof-of-stake systems require capital-at-risk to prevent a logical/optimal strategy of multiple chain voting. We intend to implement 'slashing' rules which, if broken, result some amount of the offending validator's deposited stake to be removed from circulation. Given the ordering properties of the PoH data structure, we believe we can simplify our slashing rules to the level of a voting lockout time assigned per vote.  
+
+I.e. Each vote has an associated lockout time (PoH duration) that represents a duration by any additional vote from that validator must be in a PoH that contains the original vote, or a portion of that validator's stake is slashable. This duration time is a function of the initial vote PoH count and all additional vote PoH counts.  It will likely take the form:
+
+Lockout<sub>i</sub>(PoH<sub>i</sub>, PoH<sub>j</sub>) = PoH<sub>j</sub> + K * exp((PoH<sub>j</sub> - PoH<sub>i</sub>) / K)
+
+Where PoH<sub>i</sub> is the height of the vote that the lockout is to be applied to and PoH<sub>j</sub> is the height of the current vote on the same branch. If the validator submits a vote on a different PoH branch on any PoH<sub>k</sub> where k > j > i and PoH<sub>k</sub> < Lockout(PoH<sub>i</sub>, PoH<sub>j</sub>), then a portion of that validator's stake is at risk of being slashed.
```

Comment:
A breath of fresh air to see the notation explained just after it's first used. Thanks so much!